lib/util/clone.ts

```diff
@@ -1,5 +1,5 @@
 import safeStringify from 'fast-safe-stringify';
 
-export function clone<T>(input: T): T {
+export function clone<T>(input: T = null): T {
```

Comment:
Why do we need null here?

Reply:
Because we have a lot of tests where we mock body but don't mock headers.. and JSON.parse works with `null` but not `undefined`